Sources/RealDeviceMapLib/Webhook/WebHookRequestHandler.swift

```diff
@@ -1084,9 +1084,11 @@ public class WebHookRequestHandler {
         if device == nil || questArTargetMap.isEmpty() {
             return true
         }
-        let targetMode = questArTargetMap.getValueAt(key: device!, time: timestamp) ?? true
```

Comment:
if no entry for device in ArTargetMap exists it's a non-RDM device which is sending data to RDM